governance/contracts/ValidatorSet.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity ^0.5.0;
+
+import "./Ownable.sol";
+
+contract ValidatorSet is Ownable {
+
+  mapping(bytes32 => address) internal _validatorsAddress;
```

Comment:
1. Not sure the usage of underscore to prefix local variable is a bad practice. I actually take from [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/ownership/Ownable.sol) as you can see here.
2. True, it would be good to rename it to `_validatorAddresses`
3. Actually TMT pub keys takes 40 chars and that was the reason to use a sha512 to generate an unique key for them which fits into 32 bytes.

Let's have another offline chat regarding changes on this SC